.github/workflows/alerts-deploy.yml

```diff
@@ -25,5 +26,5 @@ jobs:
         env:
           APIKEY: ${{ secrets.NAIS_DEPLOY_APIKEY }}
           CLUSTER: prod-fss
-          RESOURCE: alerts.yml
+          RESOURCE: alerts-prod.yml
```

Comment:
Jeg synes vi skal strebe for å ha de samme alertsa i produksjon og test. På sikt bør vi ha mulighet til å teste de i test først man prod-setter de og.

I tillegg er jo filene identiske?

Reply:
De er ikke helt like. linken til logs.adeo.no skiller på Q og P, som var grunnen til at man trenger 2 filer